dist/fabric.js

```diff
@@ -25203,8 +25203,8 @@ fabric.Image.filters.BaseFilter.fromObject = function(object, callback) {
      * @param {Number} char the character number
      * @return {Number} fontSize of the character
      */
-    getHeightOfChar: function(line, char) {
-      return this.getValueOfPropertyAt(line, char, 'fontSize');
+    getHeightOfChar: function(line, _char) {
+      return this.getValueOfPropertyAt(line, _char, 'fontSize');
```

Comment:
should also fix the JSDOC 3 lines above

Reply:
ok. LEts do that. Will you backport it and build all of 3.0.1 3.1.1 and 3.2.1 ?